### PR TITLE
Java doc update

### PR DESCRIPTION
**Pull request by zzybluebell**
Java doc update with minor changes in "." and description

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/EditReservationCommand.java [hunk 1]
```diff
@@ -158,27 +190,46 @@ public EditReservationDescriptor(EditReservationDescriptor toCopy) {
...
 
+        /**
+         * Gets remark.
+         */
         public Optional<Remark> getRemark() {
             return Optional.ofNullable(remark);
         }
 
+        /**
+         * Override the equal method.
```
Comment (HolmesJJ) [inline src/main/java/seedu/address/logic/commands/EditReservationCommand.java:231]: ni ge gou bi

File: src/main/java/seedu/address/logic/commands/EditReservationCommand.java [hunk 2]
```diff
@@ -158,27 +190,46 @@ public EditReservationDescriptor(EditReservationDescriptor toCopy) {
...
+         * Sets DateTime.
+         *
+         * @param dateTime the date time for editing.
+         */
         public void setDateTime(DateTime dateTime) {
             this.dateTime = dateTime;
         }
 
+        /**
+         * Gets dateTime.
```
Comment (HolmesJJ) [inline src/main/java/seedu/address/logic/commands/EditReservationCommand.java:210]: ni ge gou bi

**Review by HolmesJJ — COMMENTED** [review]
Nice

File: src/main/java/seedu/address/logic/commands/EditReservationCommand.java (hunk 1 above)
Comment (zzybluebell) [reply src/main/java/seedu/address/logic/commands/EditReservationCommand.java:231]: fixed

File: src/main/java/seedu/address/logic/commands/EditReservationCommand.java (hunk 2 above)
Comment (zzybluebell) [reply src/main/java/seedu/address/logic/commands/EditReservationCommand.java]: fixed

**Comment by zzybluebell** [comment]
All the java doc update

**Pull request merged**